frontend/vite.config.ts

```diff
@@ -108,6 +112,17 @@ export default defineConfig(({ mode }) => {
 		},
 		css: {
 			devSourcemap: true,
+			preprocessorOptions: {
+				less: {
+					javascriptEnabled: true,
+					modifyVars: {
+						hack: `true; @import "${join(
+							__dirname,
+							'src/style/AntDesign/antd.overrides.less',
+						)}";`,
+					},
+				},
+			},
```

Comment:
@lewisl9029 i put this all back because vite dev locally would not have antd styling otherwise. should be ok, right?

Reply:
yeah, reflame preview is good without this change (isn't affected by vite.config.ts), but `vite dev` didn't have the antd styling without both of these changes in the vite.config.ts

Reply:
![Screenshot from 2023-11-30 19-07-00](https://github.com/highlight/highlight/assets/1351531/671f0e34-34a4-47c6-92f2-5037c2e32f9b)

hmmm

Reply:
this is actually how main looks right now for me, running with `./run-frontend.sh`

Reply:
Sorry took a while, but finally managed to repro. Had to comment out [these lines](https://github.com/lewisl9029/highlight/blob/ab02ba0554ffa62385cec6171d9b9aca82a087e6/docker/env.sh#L41-L48) for run-backend.sh to work properly after running start-infra.sh

Looking into it now.

Reply:
Wow, turns out, vite-plugin-imp has a preset option for antd to add imports to their css entry points:
![Screenshot 2023-11-30 at 8 31 22 PM](https://github.com/highlight/highlight/assets/6934200/bebb20cb-b678-4af6-bdd5-37e800ef4ab1)

And these overwrote classes from our prebuilt css. 🤦 

Looking at the list of libraries with built-in support, I don't think we needed it for anything other than antd previously, and we no longer need it for antd thanks to the prebuilt stylesheet, so I ended up just removing it altogether in https://github.com/highlight/highlight/pull/7255/commits/ebb92c7f65db0182958e2d7e289de40f13c2790b. 

Longer term we might add something similar natively in Reflame to help users map named imports to specific entry points automatically to better optimize their prod bundles.

Reply:
wow, interesting! thanks for getting to the bottom of this. looks good with your changes locally, merging!